services/checks/checks.go

```diff
@@ -375,29 +447,46 @@ func (s *Service) minPMMAgentVersion(t check.Type) *version.Parsed {
 }
 
 // executeChecks runs all available checks for all reachable services.
-func (s *Service) executeChecks(ctx context.Context) {
+func (s *Service) executeChecks(ctx context.Context) error {
 	s.l.Info("Executing checks...")
 
+	disabledChecks, err := s.GetDisabledChecks()
+	if err != nil {
+		return err
+	}
+
 	var checkResults []sttCheckResult
 
-	mySQLCheckResults := s.executeMySQLChecks(ctx)
+	mySQLCheckResults := s.executeMySQLChecks(ctx, disabledChecks)
 	checkResults = append(checkResults, mySQLCheckResults...)
 
-	postgreSQLCheckResults := s.executePostgreSQLChecks(ctx)
+	postgreSQLCheckResults := s.executePostgreSQLChecks(ctx, disabledChecks)
 	checkResults = append(checkResults, postgreSQLCheckResults...)
 
-	mongoDBCheckResults := s.executeMongoDBChecks(ctx)
+	mongoDBCheckResults := s.executeMongoDBChecks(ctx, disabledChecks)
 	checkResults = append(checkResults, mongoDBCheckResults...)
 
 	s.alertsRegistry.set(checkResults)
+
+	return nil
 }
 
 // executeMySQLChecks runs MySQL checks for available MySQL services.
-func (s *Service) executeMySQLChecks(ctx context.Context) []sttCheckResult {
+func (s *Service) executeMySQLChecks(ctx context.Context, except []string) []sttCheckResult {
```

Comment:
This `except` field is not clear.
Can we receive the list of checks and a function named `except` or something else? Or using a struct so we can see the named parameters?
Maybe we can create a checks type, which would be an []string and we could attach an `except` method to it.